backend/app/router.js

```diff
@@ -206,6 +206,9 @@ router.route('/:realm/v0.2/tasks/:id')
     .put(authenticate('jwt').always, jsonParser, tasks.updateOne)
     .delete(authenticate('jwt').always, tasks.delete);
 
+router.route('/:realm/v0.2/tasks-with-discussions/:id')
```

Comment:
I'm not wild about this endpoint name because it makes it sound like we're just fetching tasks with discussions only?